lib/Model.js

```diff
@@ -78,7 +84,7 @@ class Model {
     failedLevelsFound.sort();
 
     const advisoriesFound = [
-      ...new Set(this.advisoriesFound.map(advisoryMapper)),
+      ...new Set(this.advisoriesFound.map(advisoryMapper).filter(Boolean)),
```

Comment:
Out of curiosity, why is this `.filter(Boolean)` necessary?

Reply:
Not really necessary, so I deleted it.